src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -45,6 +45,7 @@ class UseCaseID(Enum):
     UseCaseID.SESSIONS: "sentry-metrics.cardinality-limiter.limits.releasehealth.per-org",
     UseCaseID.SPANS: "sentry-metrics.cardinality-limiter.limits.spans.per-org",
     UseCaseID.CUSTOM: "sentry-metrics.cardinality-limiter.limits.custom.per-org",
+    UseCaseID.PROFILES: "sentry-metrics.cardinality-limiter.limits.profiles.per-org",
```

Comment:
Can you explain what this actually limits? And what the impact will be?

Reply:
> number of metrics

Just to clarify a bit, not metrics but distinct time series in storage.

Reply:
Do we have any way of checking if we've exceeded this limit? And debugging why it's being exceeded.

Reply:
@Zylphrex Yes!

We have currently 2 cardinality limiters, the Python one is disabled though. Relay has some metrics but also emits a Sentry error (sampled) for additional details.